sdk/js/src/algorand/Algorand.ts

```diff
@@ -369,6 +374,8 @@ export function _parseVAAAlgorand(vaa: Uint8Array): ParsedVAA {
   //    ret.len=vaa.slice(off).length)
   //    ret.act=buf.readIntBE(off, 1))
 
+  ret.Body = vaa.slice(off);
+
```

Comment:
You added TokenId? and uri? to the ParsedVAA type.  Can you set either of those here?

Reply:
Nevermind.  Those are set in the NFT version.